src/test/java/io/penguinstats/controller/TestReportController.java

```diff
@@ -31,52 +31,45 @@
 @FixMethodOrder(MethodSorters.NAME_ASCENDING)
 public class TestReportController {
 
-    private static String itemDropHashId;
+	private static String itemDropHashId;
 
-    @Autowired
-    MockMvc mockMvc;
+	@Autowired
+	MockMvc mockMvc;
 
-    @AfterClass
-    public static void cleanDB() {
-        TestConfig.mongoTemplate.getCollection("user").deleteMany(new Document());
-        TestConfig.mongoTemplate.getCollection("item_drop_v2").deleteMany(new Document());
-    }
+	@AfterClass
+	public static void cleanDB() {
+		//        TestConfig.mongoTemplate.getCollection("user").deleteMany(new Document());
```

Comment:
why?

Reply:
in current development, i copied all data from prod into my local, since i want to see how it performs using real data without affecting prod.